client/src/components/Joystick.tsx

```diff
@@ -0,0 +1,92 @@
+import { Joystick } from 'react-joystick-component'
+
+export interface JoystickMovement {
+  isMoving: boolean
+  direction: Direction
+}
+
+type Direction = {
```

Comment:
let's use interface instead of type alias here

Reply:
Ok 👍